tests/test_pipeline_mixin.py

```diff
@@ -504,6 +504,9 @@ def validate_test_components(test_case, task, model, tokenizer, processor):
         # For CLIP-like models
         if config_vocab_size is None and hasattr(model.config, "text_config"):
             config_vocab_size = getattr(model.config.text_config, "vocab_size", None)
+        elif config_vocab_size is None and hasattr(model.config, "text_encoder"):
+            config_vocab_size = getattr(model.config.text_encoder, "vocab_size", None)
```

Comment:
Try to get the vocab size for musicgen (tiny) model

Reply:
For music-get the input/output vocab sizes are different:
* Input vocab size is the vocab size of the text encoder model
* Output vocab size is the number of vq-codes the audio decoder can generate. These vq-codes then go through the codec model to recover the final waveform
* Thus, the actual output is **continuous** (a 1-d audio waveform)

Either way, I don't think the config vocab size is required for the TTA test: https://github.com/huggingface/transformers/blob/fc296f419e451d3c595a68455f18e20a3819fcd5/tests/pipelines/test_pipelines_text_to_audio.py#L180